### ISSUE TITLE
Incorrect video duration time in XP

### ISSUE THREAD

**Issue by avaxx** [gpac/gpac#1469]
By muxing YouTube video and audio streams in XP SP2 or SP3 with MP4Box from GPAC v0.8.0 sometimes I get videos which report double duration time. In some cases that can be solved by changing full path length of MP4Box.exe, setting it to run in Win98 compatibility mode, by setting duration time of video stream... but there are no rules, all this may or may not work. It's even possible to get one video with correctly recorded duration time and another with doubled one by using the same command line for the same streams in different command prompts?! I haven't noticed any problem in Win7. I wonder what causes the problem and if there is some simple workaround?

**Comment by jeanlf** [comment]
could you upload the input files you are trying to mux? Can you reproduce this with 0.9.0 ?

**Comment by avaxx** [comment]
Just to mention I had to change MajorSubsistemVersion of Mp4box.exe to be able to start it in XP (I set it to 5 by using CFF Explorer, some kernel update would help also). Yes there is the problem with v0.9.0 as well. Here are some files for testing: https://app.box.com/s/yigu9d1agrst913vdwvefhqegtaqsjai. There are cases when problem occurs by adding only video. In case of using a command line it would be interesting to try with or without full path for video, with or wihtout "#video" or "#audio", it's strange but sometimes such details affect the result. Btw I even tried with some Kernel32.dll wrappers for XP which add some new API functions and use original Kernel32.dll, for some reason XomPie makes things better, especially with Yamb, but not always. Thank you very much for your effort!
